modules/administration_manual/pages/configuration/files/external_storage/smb.adoc

```diff
@@ -1,48 +1,58 @@
-`php-smbclient`.
-
-You also need the Samba client installed on your Linux system. This is
-included in all Linux distributions; on Debian, Ubuntu, and other Debian
-derivatives this is `smbclient`. On SUSE, Red Hat, CentOS, and other Red
-Hat derivatives it is `samba-client`. You also need `which` and
-`stdbuf`, which should be included in most Linux distributions.
-
-You need the following information:
-
-* Folder name for your local mountpoint.
-* Host: The URL of the Samba server.
-* Username: The username or domain/username used to login to the Samba
-server.
-* Password: the password to login to the Samba server.
-* Share: The share on the Samba server to mount.
-* Remote Subfolder: The remote subfolder inside the Samba share to mount
-(optional, defaults to /). To assign the ownCloud logon username
-automatically to the subfolder, use `$user` instead of a particular
-subfolder name.
-* And finally, the ownCloud users and groups who get access to the
-share.
-
-Optionally, you can specify a `Domain`. This is useful in cases where
-the SMB server requires a domain and a username, and an advanced
-authentication mechanism like session credentials is used so that the
-username cannot be modified. This is concatenated with the username, so
-the backend gets `domain\username`
+= Samba File Server Configuration (SMB/CIFS)
+// URLs
+:libsmbclientphp-url: https://github.com/eduardok/libsmbclient-php
+:smbclient-url: https://www.samba.org/samba/docs/man/manpages-3/smbclient.1.html
+:smbcifs-url: https://docs.microsoft.com/en-us/windows/desktop/FileIO/microsoft-smb-protocol-and-cifs-protocol-overview
+
+ownCloud can connect to Windows file servers, and other SMB-compatible (Samba) servers, by using the {smbcifs-url}[SMB/CIFS] backend.
+
+== Dependencies
+
+There are two dependencies to connect to your ownCloud installation to Windows file servers, 
+
+. xref:libsmbclient-php-or-smbclient[libsmbclient-php or smbclient]
+. xref:the-samba-client[The Samba Client]
+
+=== libsmbclient-php or smbclient
+
+ownCloud's SMB/CIFS backend requires either {libsmbclientphp-url}[the libsmbclient-php module] (version 0.8.0+) or {smbclient-url}[the smbclient command] (and its dependencies) to be installed on the ownCloud server. 
+We highly recommend `libsmbclient-php`, but it isn't required. 
+If it is installed smbclient won't be needed. 
+Most Linux distributions provide `libsmbclient-php` and typically name it `php-smbclient`.
+
+=== The Samba Client
+
+The Samba client must be installed on your Linux system. 
+It is included in all Linux distributions; on _Debian_, _Ubuntu_, and other Debian
+derivatives this is `smbclient`. 
+On _SUSE_, _Red Hat_, _CentOS_, and other Red Hat derivatives it is `samba-client`. 
+You also need `which` and `stdbuf`, which should be included in most Linux distributions.
+
+== Configuration
+
+When configuring it, you will need the following information:
+
+* The folder name for your local mountpoint.
```

Comment:
Proposal:
A name for your local mountpoint which shows up as a folder

Reply:
On thinking about it further, "The folder name, which will be your local mount point." will work better.